examples/file-associations/src-tauri/src/main.rs

```diff
@@ -29,6 +29,9 @@ fn handle_file_associations(app: AppHandle, files: Vec<PathBuf>) {
 fn main() {
   tauri::Builder::default()
     .setup(|app| {
+      #[cfg(not(any(windows, target_os = "linux")))]
+      let _app = app;
```

Comment:
this complicates the example a bit, I'd rather just add `#![allow(unused)]` at the top

Reply:
i can add it inline.. i hate suppressing warns globally ;D